Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonForm.cs

```diff
@@ -101,6 +101,7 @@ public FormFixedButtonSpecCollection(KryptonForm owner)
         private bool _mdiTransferred;
         private Control? _activeControl;
         private KryptonFormTitleStyle _titleStyle;
+        private KryptonThemedSystemMenu? _themedSystemMenu;
```

Comment:
Why here and not in the VisualForm base class?

Reply:
Now using a service